docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/3122_insufficient_capacity/3122_4.md

```diff
@@ -0,0 +1,30 @@
+# Run jobs in case of insufficient capacity for region without Run shift policy flag
+
+Test verifies that run isn't restarted in another regions in case of insufficient capacity if initial region doesn't have Run shift policy flag.
+
+**Prerequisites**:
+- Admin user
+- Perform [3122_1](3122_1.md) case
+
+**Preparations**
+1. Login as admin user from the prerequisites
+2. Open the **Settings** page
+3. Click the **CLOUD REGIONS** tab
+4. Select `region1` on the left panel
+5. Save the ***Run shift policy*** checkbox state, then untick it if needed
+
+| Steps | Actions | Expected results |
+| :---: | --- | --- |
+| 1 | Open the Tools page | |
+| 2 | Select test tool | |
+| 3 | At the tool page, hover over the **Run v** button | |
+| 4 | Click the **Custom settings** button in the list | |
+| 5 | Expand the **Exec environment** section | |
+| 6 | Select `p2.8xlarge` value for **Node type** | |
+| 7 | Select `region1` value for **Cloud Region** | |
+| 8 | Expand the **Advanced** section | |
+| 9 | Select `On-demand` value for **Price type** | |
+| 10 | Launch the run | |
+| 11 | At the Runs page, click the just-launched run | |
+| 12 | Wait until ***RestartPipelineRun*** task appears | <li> Run is stopped <li> Message `Server failure reason: Insufficient instance capacity.` appears under the run title |
+| 13 | Click ***RestartPipelineRun*** task | Records `Run '<runID>' failed to start. <br> Region relaunch is not available for runs for region '<region1>'.` are shown in the log |
```

Comment:
Add below:

After the test, open the **CLOUD REGIONS** tab of the **Settings** page and return states to the ***Run shift policy*** checkboxes for all regions saved at steps 4-9 of the preparations of case [3122_1](3122_1.md)

Reply:
Done